app/pages/sections/catalogue-solutions/order-item/flat/declarative/ui-tests/withSavedData.ui.test.js

```diff
@@ -29,7 +29,7 @@ const orderItem = {
     name: 'month',
     description: 'per month',
   },
-  price: 0.1,
+  price: 0.11,
```

Comment:
I would keep this price as 0.1 and make sure it shows up as 0.10 in your expectation